views/four-o-four.css

```diff
@@ -0,0 +1,106 @@
+
+html{
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
+}
+html,
+body,
+.four_oh_four {
+    padding: 0px;
+    margin: 0px;
+    height: 100%;
+    width: 100%;
+}
+.four_oh_four {
+    background-image: -o-linear-gradient(-38deg, #1AB5C2 0%, #0572E0 100%);
+    background-image: -moz-linear-gradient(-38deg, #1AB5C2 0%, #0572E0 100%);
+    background-image: -ms-linear-gradient(-38deg, #1AB5C2 0%, #0572E0 100%);
+    background-image: linear-gradient(128deg, #1AB5C2 0%, #0572E0 100%);
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    color: #fff;
+}
+.four_oh_four .logo {
+    display: block;
+    margin: 0 auto;
+    top: 50px;
+    position: relative;
+}
+.four_oh_four .nav {
+    list-style: none;
+    font-weight: 200;
+    font-size: 18px;
+    margin: 0 auto;
+    position: absolute;
+    padding: 0;
+    bottom: 50px;
+    width: 100%;
+    text-align: center;
+}
+.four_oh_four .nav li {
+    display: inline-block;
+    margin: 0px 10px;
+}
+.four_oh_four .nav a {
+    color: #fff;
+    text-decoration: none;
+}
+.four_oh_four .nav a:hover {
+    text-decoration: underline;
+}
+.four_oh_four .error {
+    display: block;
+    position: absolute;
+    top: 50%;
+    -moz-transform: translateY(-50%);
+    -webkit-transform: translateY(-50%);
+    transform: translateY(-50%);
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    width: 100%;
+}
+.four_oh_four h1 {
+    font-weight: 100;
+    font-size: 240px;
+    margin: 0;
+    padding: 0;
+}
+.four_oh_four h2 {
+    font-weight: 200;
+    font-size: 28px;
+    margin: 0;
+    padding: 0;
+}
+.four_oh_four #emoji {
+    position: relative;
+    top: 40px;
+    background-image: url('https://www.parse.com/images/404/sprite.png');
```

Comment:
We may want to host the image within parse-served itself maybe? Chances are in a years time this image URL probably won't work?